### DISCUSSION TITLE
Example files from Radiance tutorial to do some how-to documentation.

### DISCUSSION THREAD

**Discussion by sariths** [LBNL-ETA/pyradiance#23] [General]
@taoning I am wondering if it makes sense to use the files from the matrix tutorial for documenting some examples. The files are here: https://github.com/sariths/radTutorialFiles. The three directories in that repo contain input (and empty output) directories for pretty much every use case for standard Radiance.
I would prefer adding them to have them stored locally in the assets folder, but if storing them here is an issue, we can point to the existing repository.

**Comment by taoning** [comment]
That's a great idea. I think we can store those in the `doc/assets`.
